src/AppBundle/Exception/RuntimeException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace AppBundle\Exception;
+
+class RuntimeException extends \RuntimeException
```

Comment:
This class is needless, let's make the others extend `\RuntimeException` instead, what do you think?